### ISSUE TITLE
util.js: refactor, named exports, explicit named imports

### ISSUE THREAD

**Issue by backspaces** [backspaces/asx#19]
The util.js module historically was where everything went that didn't fit in a particular Module. It has grown to be twice the size of other modules .. here's a partial listing by size:
```
-rw-r--r--@ 1 owen  staff    33K Aug 11 13:55 util.js
-rw-r--r--@ 1 owen  staff    15K Jun  1 18:00 DataSet.js
-rw-r--r--@ 1 owen  staff    14K Aug  8 13:24 Patches.js
-rw-r--r--@ 1 owen  staff    11K Jul 24 17:28 ColorMap.js
-rw-r--r--@ 1 owen  staff    11K Aug 11 13:38 Turtle.js
-rw-r--r--@ 1 owen  staff    10K Sep 25 13:34 ThreeMeshes.js
-rw-r--r--@ 1 owen  staff   9.6K Aug  9 16:38 SpriteSheet.js
-rw-r--r--@ 1 owen  staff   8.6K Aug  9 16:41 Color.js
-rw-r--r--@ 1 owen  staff   7.8K Sep 26 13:37 Model.js
       ... etc
```
It's time to toss functions no longer needed or used only once. Then refactor parts into smaller modules and/or convert to named exports so that only what is needed is imported.

It will touch most of the other modules and test models but in an easy to manage way.

Initially, we'll simply replace: `import util from './util.js'` to `import * as util from './util.js` but then eventually be clearer on the imports: `import {foo, bar, baz} from './util.js'`

This will significantly reduce the size of the utilities, clarify the use of them, and create a few simpler modules like `Debug.js`, `Canvas.js`, and move array utils into `AgentArray.js`, the core array module for `AgentSet.js`
